### PR TITLE
Refacto metro plan styles

### PR DESCRIPTION
**Pull request by KamiAmiga**

### CODE REVIEW

File: client/app/app.tpl.less [hunk 1]
```diff
@@ -74,6 +74,11 @@
 
 @label-default-bg: @gray;
 
+//oui-colors
+@mischka: #D5DAE0;
```
Comment (antleblanc) [inline client/app/app.tpl.less]: Any reason to don't use these variables?

```less
@oui-color-mischka
@oui-color-shuttle
@oui-color-foam
```

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference.html [hunk 2]
```diff
@@ -1,182 +1,185 @@
...
-                    </i>
+                    </span>
 
-                    <i class="fa fa-deaf left-space-m8 vertical-middle"
+                    <span class="fa fa-deaf fs16 left-space-m8"
                        aria-hidden="true"
                        data-ng-if="!participant.hear"
                        data-translate-attr="{ title: 'telephony_number_feature_conference_participants_state_deaf' }">
-                    </i>
+                   </span>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference.html]: indentation

**Comment by frenautvh** [comment]
@KamiAmiga PR into develop branch

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference.html [hunk 3]
```diff
@@ -1,182 +1,185 @@
...
-                    </i>
+                        data-jsplumb-connection-target="incoming-call-{{ $ctrl.numberCtrl.number.serviceName }}"
+                        aria-hidden="true">
+                    </span>
                 </div>
-                <div class="anchor-content service-presentation"
-                     data-translate="telephony_number_feature_conference_participants_emprty">
+                <div class="voip-plan__step-description">
+                     <span class="voip-plan__step-info"
+                           data-translate="telephony_number_feature_conference_participants_emprty"></span>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference.html]: w00t :innocent:! This is my fault but `emprty` => `empty`?

**Comment by Jisay** [comment]
I close this PR. This will be merged with a complete refacto of dialplans code.

**Pull request closed without merging**